src/services/openletters/index.jsx

```
@@ -9,20 +11,16 @@ import ErrorMessage from "../../components/form/ErrorMessage";
 import Form from "../../components/form/Form";
 import { Spinner } from "../../components/theme/Loader";
 import AppTextArea from "../../components/form/AppTextArea";
-import { FaPlusCircle } from "react-icons/fa";
+import { FaPlusCircle, FaRegCheckCircle } from "react-icons/fa";
 import { Toast } from "../../components/theme/Toast";
```

ESLint

[error] 'Toast' is defined but never used.

(no-unused-vars)